.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+<!--
+  All PRs must be labelled with one of the following labels:
```

Comment:
What about updating README or other documentation, use "chore"?

Reply:
Yes that is what that label is for, it will go under "Maintenance" on the change-log.

Reply:
Other documentation should be put under `area/documentation` I will add that to the set of labels before merging this.

Reply:
Just reminding about adding `documentation` ^